libra-node/src/main_node.rs

```diff
@@ -120,11 +120,11 @@ pub fn setup_network(
             ProtocolId::from_static(CONSENSUS_RPC_PROTOCOL),
             ProtocolId::from_static(ADMISSION_CONTROL_RPC_PROTOCOL),
         ]);
-    if config.is_permissioned {
-        // If the node wants to run in permissioned mode, it should also have authentication and
-        // encryption.
+    if config.enable_remote_authentication {
+        // If the node wants to run in enable_remote_authentication mode, it should also use noise for
+        // authentication and encryption.
         assert!(
-            config.enable_encryption_and_authentication,
+            config.enable_noise,
```

Comment:
better to assert in the config code so that we can validate configs in the config code only

Reply:
As discussed offline, we'll address these logic changes in a future PR. This PR is just for renaming.